packages/discord.js/typings/index.d.ts

```diff
@@ -1979,6 +1987,37 @@ export class BaseInteraction<Cached extends CacheType = CacheType> extends Base
   public isRepliable(): this is RepliableInteraction<Cached>;
 }
 
+export class InteractionCallback {
+  public constructor(client: Client<true>, data: RESTAPIInteractionCallbackObject);
+  public activityInstanceId: string | null;
+  public readonly client: Client<true>;
+  public get channel(): TextBasedChannel | null;
+  public channelId: Snowflake | null;
+  public get createdAt(): Date;
+  public get createdTimestamp(): number;
+  public get guild(): Guild | null;
+  public guildId: Snowflake | null;
+  public id: Snowflake;
```

Comment:
`guildId` was removed (111b3ddde9bca7f5a0df8f096991ba6ef68d4bd0 ).

```suggestion
  public get createdTimestamp(): number;
  public id: Snowflake;
```